src/main/res/values/strings.xml

```diff
@@ -78,24 +78,24 @@
     <string name="link_copied_to_clipboard">Link copied to clipboard</string>
     <string name="links">Links</string>
     <string name="list_animation">List Animations</string>
-    <string name="list_animation_description">Show list animation on the main page</string>
+    <string name="list_animation_description">Animation the list on the main page</string>
     <string name="merging_FORMAT">Merging %s</string>
     <string name="name">Name</string>
     <string name="network_error_DESC">Network error.</string>
     <string name="never">Never</string>
-    <string name="new_updates_available">New versions of applications available</string>
+    <string name="new_updates_available">New versions available</string>
     <plurals name="new_updates_DESC_FORMAT">
-        <item quantity="one">%d application has a new version.</item>
-        <item quantity="other">%d applications with new versions.</item>
+        <item quantity="one">%d app has a new version.</item>
+        <item quantity="other">%d apps with new versions.</item>
     </plurals>
-    <string name="no_applications_available">No available applications</string>
-    <string name="no_applications_installed">No installed applications</string>
-    <string name="no_description_available_DESC">No description available.</string>
-    <string name="no_matching_applications_found">Could not find any such applications</string>
+    <string name="no_applications_available">None available</string>
+    <string name="no_applications_installed">None installed</string>
```

Comment:
?

Reply:
Doesn't make sense for me

Reply:
I would suggest changing these

Reply:
I think something like "Install an app that can ___ first" makes sense.